gulpfile.js

```diff
@@ -88,7 +88,11 @@ gulp.task('build:styles', () => {
 gulp.task('build:standalone', () => {
   const prettyJs = gulp.src('dist/sweetalert2.js')
   const prettyCssAsJs = gulp.src('dist/sweetalert2.css')
```

Comment:
Should we just do this
```suggestion
   const prettyCssAsJs = gulp.src('dist/sweetalert2.min.css')
```
instead of doing the same ("uglifying" css) with `css2js` below?